rhel/vex/updater.go

```diff
@@ -99,6 +101,7 @@ func (f *Factory) Configure(ctx context.Context, cf driver.ConfigUnmarshaler, c
 type Updater struct {
 	url             *url.URL
 	client          *http.Client
+	version         string
```

Comment:
Why is this a runtime value and not a package constant?

Reply:
It doesn't need to be, I just thought it made sense to tie the version to the thing it's describing, but same same, I can just read the const.

Reply:
I'd rather it be a constant (like the indexers) because it can't vary at runtime.